rollup.config.js

```diff
@@ -16,7 +16,7 @@ export default {
     },
     {
       file: pkg.module,
-      format: 'es'
+      format: 'esm'
```

Comment:
> `es` is not the best word to identify es modules. It's used by
`material-ui` library to distribute es6+ code. `esm` is a better
alternative which is used by awesome esm loader. In the latest release
rollup started to adrop `esm` word instead of `es`, for now just as an
alias.

This seems to be ... somewhat subjective. Even the related Rollup issue (rollup/rollup/issues/1917) says they'll add "esm" as an alias for "es" but they aren't ready to make it _preferred_ or the _default_. (The [Rollup docs also show "es"](https://rollupjs.org/guide/en#configuration-files).)

I don't think we need to update this until/unless it becomes Rollup's recommended format.

Reply:
It's official api. Guys just trying to make less breaking changes. With a time `es` will be deprecated.

Reply:
Yeah, I just need to update docs.

Reply:
> With a time `es` will be deprecated.

Where is this said?

Reply:
They are open to deprecation but we need a time because option can be used by plugins.
https://github.com/rollup/rollup/pull/2102

Reply:
I think I will manage this process.

Reply:
I see. Okay then. 👍